fhir-server/src/test/java/au/csiro/pathling/test/UnitTestDependencies.java

```diff
@@ -38,7 +61,18 @@ class UnitTestDependencies {
   static SparkSession sparkSession(@Nonnull final Configuration configuration,
       @Nonnull final Environment environment,
       @Nonnull final Optional<SparkListener> sparkListener) {
-    return Spark.build(configuration, environment, sparkListener);
+    final List<SqlFunction1> sqlFunction1 = List.of(new CodingToLiteral());
+    final List<SqlFunction2> sqlFunction2 = List.of(new DateTimeAddDurationFunction(),
+        new DateTimeSubtractDurationFunction(), new DateAddDurationFunction(),
+        new DateSubtractDurationFunction(),
+        new DateTimeEqualsFunction(), new DateTimeGreaterThanFunction(),
+        new DateTimeGreaterThanOrEqualToFunction(), new DateTimeLessThanFunction(),
+        new DateTimeLessThanOrEqualToFunction(), new TimeEqualsFunction(),
+        new TimeGreaterThanFunction(), new TimeGreaterThanOrEqualToFunction(),
+        new TimeLessThanFunction(), new TimeLessThanOrEqualToFunction());
+    final List<SqlFunction3> sqlFunction3 = List.of(new TemporalDifferenceFunction());
+    return Spark.build(configuration, environment, sparkListener, sqlFunction1, sqlFunction2,
+        sqlFunction3);
   }
```

Comment:
All these functions can probably be found with 'ListableBeanFactory` and I believe there should be a way to get access to it in this method (normally this would be with the 'BeanFactoryAware').

Reply:
Fixed.